WordPress/src/main/java/org/wordpress/android/ui/mysite/items/SiteItemsBuilder.kt

```diff
@@ -138,17 +148,4 @@ class SiteItemsBuilder @Inject constructor(
             )
         } else emptyList()
     }
-
-    private fun getExternalSiteItems(params: SiteItemsBuilderParams): List<MySiteCardAndItem> {
-        return listOfNotNull(
-            CategoryHeaderItem(UiStringRes(R.string.my_site_header_external)),
-            ListItem(
-                R.drawable.ic_globe_white_24dp,
-                UiStringRes(R.string.my_site_btn_view_site),
-                secondaryIcon = R.drawable.ic_external_white_24dp,
-                onClick = ListItemInteraction.create(ListItemAction.VIEW_SITE, params.onClick)
```

Comment:
👋 @AjeshRPai - Thanks for the review.  That enum is in use in MySiteViewModel (L:876)

> is intentional right?
>
Yes, it is part of design to group things visually, since they're a lot of them under **Manage** section.

Reply:
Hey @ravishanker 

Sorry for not being clear enough. What I meant here was, if you see the `MySiteViewModel (L:876)`, it's a when check for the enum of `ListItemAction` item. Since there is no list item with VIEW_SITE enum `ListItemAction`. The check for the `VIEW_SITE` enum won't be needed in `MySiteViewModel` (L:876) and we no longer need the `VIEW_SITE` enum. Does that make it clear 🤔